styles.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
- [ ] Good job making the requested changes by the previous reviewer, however, the changes have introduced a white space to the right of your design. Kindly fix this.
This is coming from the fact that you have attempted to create a mobile version using the desktop view. Before you can even start to add HTML elements to your design, you should switch from desktop view to mobile view by right-clicking on your page in the browser and selecting inspect. Check out this resource with examples and pictures on how to view the mobile version on Chrome. [Click here](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome)

Your design
![image](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/81932478/ba8853b7-8ef1-4771-9458-a1b312f6c95d)

Figma design
![image](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/81932478/d2d0588c-d4b9-4a3e-997f-c85e0e8bdc04)

#### Notice where your background image is ending and where it is ending in the given design